lte/gateway/c/oai/tasks/mme_app/mme_app_bearer.c

```diff
@@ -3857,48 +3856,34 @@ void mme_app_handle_e_rab_modification_ind(
        idx++) {
     e_rab_id_t bearer_id =
         e_rab_modification_ind->e_rab_to_be_modified_list.item[idx].e_rab_id;
-    if ((current_bearer_p =
-             mme_app_get_bearer_context(ue_context_p, bearer_id)) == NULL) {
-      // useless, already tested few lines above
-      OAILOG_ERROR_UE(
-          LOG_MME_APP, ue_context_p->emm_context._imsi64,
-          "Bearer Contex for bearer_id %d does not exist for ue_id %d\n",
-          bearer_id, ue_context_p->mme_ue_s1ap_id);
-    } else {
-      s11_modify_bearer_request->bearer_contexts_to_be_modified
-          .bearer_contexts[idx]
-          .eps_bearer_id =
-          e_rab_modification_ind->e_rab_to_be_modified_list.item[idx].e_rab_id;
-      s11_modify_bearer_request->bearer_contexts_to_be_modified
-          .bearer_contexts[idx]
-          .s1_eNB_fteid.teid =
-          e_rab_modification_ind->e_rab_to_be_modified_list.item[idx]
-              .s1_xNB_fteid.teid;
 
-      memcpy(
-          &s11_modify_bearer_request->bearer_contexts_to_be_modified
-               .bearer_contexts[idx]
-               .s1_eNB_fteid,
-          &e_rab_modification_ind->e_rab_to_be_modified_list.item[idx]
-               .s1_xNB_fteid,
-          sizeof(s11_modify_bearer_request->bearer_contexts_to_be_modified
-                     .bearer_contexts[idx]
-                     .s1_eNB_fteid));
-      s11_modify_bearer_request->bearer_contexts_to_be_modified
-          .bearer_contexts[idx]
-          .s1_eNB_fteid.interface_type = S1_U_ENODEB_GTP_U;
-      s11_modify_bearer_request->bearer_contexts_to_be_modified
-          .num_bearer_context++;
+    s11_modify_bearer_request->bearer_contexts_to_be_modified
+        .bearer_contexts[idx]
+        .eps_bearer_id =
+        e_rab_modification_ind->e_rab_to_be_modified_list.item[idx].e_rab_id;
 
-      OAILOG_DEBUG_UE(
-          LOG_MME_APP, ue_context_p->emm_context._imsi64,
-          "Build MBR for ue_id %d\t bearer_id %d\t enb_teid %u\t sgw_teid %u\n",
-          ue_context_p->mme_ue_s1ap_id, bearer_id,
-          s11_modify_bearer_request->bearer_contexts_to_be_modified
-              .bearer_contexts[idx]
-              .s1_eNB_fteid.teid,
-          current_bearer_p->s_gw_fteid_s1u.teid);
-    }
+    memcpy(
+        &s11_modify_bearer_request->bearer_contexts_to_be_modified
+             .bearer_contexts[idx]
+             .s1_eNB_fteid,
+        &e_rab_modification_ind->e_rab_to_be_modified_list.item[idx]
+             .s1_xNB_fteid,
+        sizeof(s11_modify_bearer_request->bearer_contexts_to_be_modified
+                   .bearer_contexts[idx]
+                   .s1_eNB_fteid));
+    s11_modify_bearer_request->bearer_contexts_to_be_modified
+        .bearer_contexts[idx]
+        .s1_eNB_fteid.interface_type = S1_U_ENODEB_GTP_U;
```

Comment:
Is this still needed considering we did a memcpy from the e_rab_modification_ind above?